src/modules/onboarding/signup/index.tsx

```diff
@@ -0,0 +1,113 @@
+import {
+  Stack,
+  Button,
+  TextField,
+  Checkbox,
+  FormControlLabel,
+  InputAdornment,
+  IconButton,
+  Typography,
+} from '@mui/material';
+import { Paper } from '../../../shared-components/Paper/Paper';
+import { styled, Theme } from '@mui/material/styles';
+import { Regular1 } from '../../../shared-components/Regular1';
+import GoogleIcon from '@mui/icons-material/Google';
+import React, { useState } from 'react';
+import Visibility from '@mui/icons-material/Visibility';
+import VisibilityOff from '@mui/icons-material/VisibilityOff';
+import Link from '@mui/material/Link';
+import ArrowBack from '@mui/icons-material/ArrowBack';
+import { Title2 } from '../../../shared-components/Title2';
+import '../signup/style.css';
+
+const StyledDiv = styled(`div`)(({ theme }: { theme: Theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'space-between',
```

Comment:
Here you need to use `justifyContent: 'flex-start',` as you don't want space between top bar and the form. 

Also, another student is working of a task that involves header element. Feel free to look here: https://github.com/frontend-from-0/montra-client/pull/8/files#diff-c9b93412afe60955680ff594a4dbb687d9618713e1ac9d048682a3fd7d70adf4

He will merge his changes soon, so you'll need to update your branch with what will be added to master so feel free to take parts that are relevant fro your task directly from the PR.

Reply:
I did it. Of course it needs some corrections, I will do it after merging.

Reply:
Anna, I also added useNavigate hook and handleClick function for Sign-up button in the Introduction page, but it doesn't work now. Could you please check it?